### DISCUSSION TITLE
chia crashes my pc

### DISCUSSION THREAD

**Discussion by dramBUe** [Chia-Network/chia-blockchain#5026] [Support]
having the chia blockchain on my computer causes my pc to crash and reboot within a few seconds of entering windows 10 over and over again. I don't even have to start the application. The only way i can stop it is to enter safemode to uninstall it then everything works fine, tested multiple times with the same results. Its my second computer with it on but my other one works fine. on the second machine i managed to get as far as linking them through my seed, syncing the blockchain and starting 2 plots then my pc crashed and started with this problem. Any help would be much appreciated. thanks

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
